keps/sig-scheduling/5007-device-attach-before-pod-scheduled/kep.yaml

```diff
@@ -39,7 +39,7 @@ milestone:
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
 feature-gates:
-  - name: DRADeviceAttachDuringScheduling
+  - name: DRAPrebindingConditions
```

Comment:
We went with "binding" not "prebinding" in the field names, so let's make this consistent and make it `DRADeviceBindingConditions`